<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModuleEditor</class>
 <widget class="QDialog" name="ModuleEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Screen Parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page1">
      <widget class="QLabel" name="labelAttrSet">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>250</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="text">
        <string>Usr Set</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelScanDot">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>250</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="text">
        <string>Scan Dot</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>80</y>
         <width>341</width>
         <height>121</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,0,5">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButtonAttrSet">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Usr Set</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonScanDot">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Scan Dot</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page2">
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>171</width>
         <height>271</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout" columnstretch="3,4">
        <item row="0" column="0">
         <widget class="QLabel" name="labelScanLines">
          <property name="text">
           <string>Scan Lines</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBoxScanLines">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="currentIndex">
           <number>3</number>
          </property>
          <item>
           <property name="text">
            <string>Static Scan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1/2 Scan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1/3 Scan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1/4 Scan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1/5 Scan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1/6 Scan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1/8 Scan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1/10 Scan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1/12 Scan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1/16 Scan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1/32 Scan</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelDataPolarity">
          <property name="text">
           <string>Data Polar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBoxWarpMode">
          <item>
           <property name="text">
            <string>Up Warp</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Down Warp</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Straight</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="comboBoxDataPolarity">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>Negative</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Positive</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="comboBoxOePolarity">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>Negative</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Positive</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelWarpMode">
          <property name="text">
           <string>Warp Mode</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelOePolarity">
          <property name="text">
           <string>OE Polar</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="labelBaseColor">
          <property name="text">
           <string>Base Color</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="comboBoxBaseColor">
          <item>
           <property name="text">
            <string>One</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Two</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Three</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="labelMirror">
          <property name="text">
           <string>Mirror</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="comboBoxMirror">
          <item>
           <property name="text">
            <string>No</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Yes</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>50</y>
         <width>171</width>
         <height>271</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="3,4">
        <item row="0" column="0">
         <widget class="QLabel" name="labelModuleLines">
          <property name="text">
           <string>Board Lines</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="labelSplit">
          <property name="text">
           <string>Split</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBoxModuleLines">
          <property name="currentIndex">
           <number>6</number>
          </property>
          <item>
           <property name="text">
            <string>1 Line</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2 Line</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4 Line</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8 Line</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10 Line</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12 Line</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>16 Line</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>32 Line</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="labelWarpTurn">
          <property name="text">
           <string>Warp Turn</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelDecodeMode">
          <property name="text">
           <string>Decode</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="comboBoxDecodeMode">
          <item>
           <property name="text">
            <string>High138</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Low138</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>HighIO</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>LowIO</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="comboBoxStrPolarity">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>Negative</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Positive</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelStrPolarity">
          <property name="text">
           <string>STR Polar</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="comboBoxSplit">
          <item>
           <property name="text">
            <string>No</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Yes</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBoxWarpBits">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>4 Bit</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8 Bit</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>16 Bit</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>32 Bit</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelWarpBits">
          <property name="text">
           <string>Warp Bits</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="comboBoxWarpTurn">
          <item>
           <property name="text">
            <string>No</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Yes</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="labelModuleName">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Module Name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditModuleName">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>10</y>
         <width>311</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonAttrSetCancel">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>340</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonAttrSetOK">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>340</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonAttrSetUnlock">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButtonAttrSetCancel</tabstop>
  <tabstop>pushButtonAttrSetOK</tabstop>
  <tabstop>lineEditModuleName</tabstop>
  <tabstop>comboBoxScanLines</tabstop>
  <tabstop>comboBoxModuleLines</tabstop>
  <tabstop>comboBoxWarpMode</tabstop>
  <tabstop>comboBoxWarpBits</tabstop>
  <tabstop>comboBoxOePolarity</tabstop>
  <tabstop>comboBoxStrPolarity</tabstop>
  <tabstop>comboBoxDataPolarity</tabstop>
  <tabstop>comboBoxDecodeMode</tabstop>
  <tabstop>comboBoxBaseColor</tabstop>
  <tabstop>comboBoxSplit</tabstop>
  <tabstop>comboBoxMirror</tabstop>
  <tabstop>comboBoxWarpTurn</tabstop>
  <tabstop>pushButtonAttrSetUnlock</tabstop>
  <tabstop>pushButtonAttrSet</tabstop>
  <tabstop>pushButtonScanDot</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
